BenchmarkTest00247.py

```
@@ -0,0 +1,12 @@
+#{fact rule=sensitive-information-leak@v1.0 defects=1}
+
+import socket
+
+
+
+# ruleid:avoid-bind-to-all-interfaces
+s = socket.socket(doesnt, matter)
```

Ruff (0.13.1)

Undefined name `doesnt`

(F821)